### ISSUE TITLE
event_observer.timeout_ms is not effective 

### ISSUE THREAD

**Issue by pc-quiknode** [stacks-network/stacks-core#5375]
**Describe the bug**
I am trying to bootstrap a Nakamoto Testnet node and keep getting this error message:
```
[testnet/stacks-node/src/event_dispatcher.rs:496] [main] Event dispatcher: connection or request failed to localhost:3700 - Custom { kind: WouldBlock, error: "Timed out while receiving request" }, backoff: 100ms, attempts: 0
```

After digging through issues of stacks-core I found there's a [PR](https://github.com/stacks-network/stacks-core/pull/5286) adding timeout_ms config to event_observer section, but it doesn't seem to be effective to resolve the issue.

**Steps To Reproduce**

Trying to bootstrap a Nakamoto Testnet follower node with stacks-blockchain-api

**Comment by jcnelson** [comment]
Correct. The stacks node must block until it receives acknowledgement that
the event was received by all observers. So the timeout was removed.

**Comment by obycode** [comment]
That timeout defines when the node will retry sending the event. It will never give up.

**Comment by obycode** [comment]
@pc-quiknode does that explain the situation you're seeing? Can we close this?
